index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Document</title>
+<link rel="stylesheet" href="style/style.css">
+</head>
+<div class="wrapper">
+<header>
+<div class="header1">
+<div class="header11">CUSTOM CARE: 888-23-4567</div>
+<div class="header12">
+<div class="US"> $ US  </div>
+<div class="Account"> <img src="img/Contact _ User.png" alt=""> MY ACCOUNT </div>

```

Comment:
class names have to follow so-called `kebab` case syntax e.g. `account`, `account-mob`. Actually there are another naming conventions but for now you have to use only simple `kebab`